pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -620,7 +619,8 @@ func (r *ReconcileChaosEngine) reconcileForDelete(request reconcile.Request) (re
 		engine.Instance.ObjectMeta.Finalizers = utils.RemoveString(engine.Instance.ObjectMeta.Finalizers, "chaosengine.litmuschaos.io/finalizer")
 		r.recorder.Eventf(engine.Instance, corev1.EventTypeNormal, "ChaosEngineStopped", " Chaos resources deleted successfully")
 	}
-	updateExperimentStatuses(engine)
+	updateExperimentStatusesForStop(engine)
```

Comment:
Can we add a detailed comment on what this achieves (i.e., setting appropriate state to ongoing and unexecuted experiments)